src/Resources/config/config.xml

```diff
@@ -200,4 +200,20 @@
             </helpText>
         </component>
     </card>
+    <card>
+        <title>Domain URL</title>
+        <input-field type="bool">
+            <name>enableOverrideDefaultDomain</name>
+            <label>Select default domain URL</label>
+        </input-field>
+        <component name="sw-entity-single-select">
+            <name>returnUrlId</name>
```

Comment:
Changing the name to `defaultDomainId` may make it more clear and consistent with the other field.